modules/core/24-host/channel_keys.go

```diff
@@ -7,6 +7,7 @@ const (
 	KeyChannelPrefix                  = "channels"
 	KeyChannelUpgradePrefix           = "channelUpgrades"
 	KeyChannelRestorePrefix           = "restore"
+  KeyUpgradePrefix                  = "upgrades"
```

Comment:
formatting is a little wack, linter is complaining!

Reply:
yea. rebase mucked it up, made it shut up now